docs/try-leverage/management-account.md

```diff
@@ -113,7 +113,7 @@ $ leverage credentials configure --type BOOTSTRAP --skip-access-keys-setup
 !!! info "More information on [`credentials configure`](/user-guide/leverage-cli/reference/credentials#configure)"
 
 ### SSO layer
-Before working on the SSO layer you have to navigate to the [AWS IAM Identity Center page](https://console.aws.amazon.com/singlesignon/) and enable Single Sign-On (SSO) by clicking on the `Enable` button.
+Before working on the SSO layer you have to navigate to the [AWS IAM Identity Center page](https://console.aws.amazon.com/singlesignon/) and enable Single Sign-On (SSO) by clicking on the `Enable` button. Additionally, to ensure a correct integration, make sure to set the primary region for SSO in AWS console to match the primary region of your Leverage project. 
```

Comment:
@rodriguez-matias Matu here I would add at the end: "e.g. `us-east-1` for both of them."

**Then:**
Additionally, to ensure a correct integration, make sure to set the primary region for SSO in AWS console to match the primary region of your Leverage project. e.g. `us-east-1` for both of them.